UI/test/integration-tests/multiSegmentSpeedLimitSpec.js

```diff
@@ -135,7 +135,7 @@ define(['chai', 'testHelpers'], function(chai, testHelpers) {
 
         expect(features).not.to.equal(0);
         _.each(features, function(feature) {
-          expect(feature.getProperties().value).to.equal(100);
+          expect(feature.getProperties().value.value).to.equal(100);
```

Comment:
Can this "value.value" be changed to something more explicit?

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code